### PR TITLE
Updated dependencies + housekeeping.

### PR DESCRIPTION
**Pull request by jeffgca**
I wanted to review the vscode plugin as-is but noticed a difference in suggested macOS installation instructions between the online guide and the README for the vscode plugin. While patching this I noticed a few a other few small things that could also be updated:

1. a dependabot alert for minimist
2. `npm run lint` spewed warnings due to a spaces / tabs mismatch
3. a few random missing semicolons.

### CODE REVIEW

**Comment by jeffgca** [comment]
...please let me know if y'all would rather this PR be split up etc.

File: tslint.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "rules": {
-    "indent": [true, "tabs"],
+    "indent": [true, "spaces"],
     "semicolon": [true, "always"]
```
Comment (expede) [inline tslint.json:4]: Given that we don't use semicolons anywhere else, we should probably bring this rule inline

File: tslint.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "rules": {
-    "indent": [true, "tabs"],
+    "indent": [true, "spaces"],
```
Comment (expede) [inline tslint.json:3]: 💯 👍

File: src/extension.ts [hunk 3]
```diff
@@ -47,21 +47,21 @@ const getOpenFileURI = () => {
     vscode.window.showErrorMessage('No editor active');
     return undefined;
   }
-}
+};
 
 /**
  * Opens a terminal and sends it a command
  */
 const sendAndFocus = (command: string) => () => {
```
Comment (expede) [inline src/extension.ts:55]: Less for this PR and more a general question: why the unit currying here? 🤔 I don't see it getting partially applied anywhere, but may be just not be seeing it

**Comment by expede** [comment]
Thanks @therealjeffg 🙌 

I haven't actually ever looked at this code before. Noted a few things as I'm working through it; may open a PR of my own shortly 😛

**Review by expede — APPROVED** [review]
Looks good to me! I haven't touched this code before, so I'll leave an actual review to @walkah or @bgins, though.

File: README.md [hunk 4]
```diff
@@ -46,8 +46,7 @@ Check the [Fission Installation Guide](https://guide.fission.codes/installation)
 
 #### MacOS
 ```bash
-brew tap fission-suite/fission
-brew install fission-cli
+brew install fission-suite/fission/fission-cli
```
Comment (bgins) [inline README.md:49]: :100::pray:

File: tslint.json (hunk 1 above)
Comment (jeffgca) [reply tslint.json:4]: There's a few options then ( and I am a little ignorant of typescript to being with ) but from what you're saying, this should be changed to "never":

```
One of the following arguments must be provided:

    "always" enforces semicolons at the end of every statement.
    "never" disallows semicolons at the end of every statement except for when they are necessary.

The following arguments may be optionally provided:

    "ignore-interfaces" skips checking semicolons at the end of interface members.
    "ignore-bound-class-methods" skips checking semicolons at the end of bound class methods.
    "strict-bound-class-methods" disables any special handling of bound class methods and treats them as any other assignment. This option overrides "ignore-bound-class-methods".
```

https://palantir.github.io/tslint/rules/semicolon/

File: src/extension.ts (hunk 3 above)
Comment (bgins) [reply src/extension.ts:55]: I was also wondering about this. It looks like `registerCommand` expects a callback: https://code.visualstudio.com/api/references/vscode-api#commands.registerCommand, and we are configuring the callback with some extra arguments.

File: tslint.json (hunk 2 above)
Comment (bgins) [reply tslint.json:3]: Spaces for sure :+1:

**Review by bgins — APPROVED** [review]
Looks good! :100: :pray:

File: tslint.json (hunk 1 above)
Comment (expede) [reply tslint.json:4]: Yup, I'm suggesting `never` 👍 But again, I've never touched this code, so I may be missing some context

File: src/extension.ts (hunk 3 above)
Comment (expede) [reply src/extension.ts:55]: Hmmm, I'd suggest wrapping in `() =>` at the call site in that case. It feels weird to need to pass this extra `()` all over the place, no?

File: tslint.json (hunk 1 above)
Comment (expede) [reply tslint.json:4]: But we can do that in a different PR

Comment (walkah) [reply tslint.json:4]: I had no idea that I was in the midst of semi-colon haters ;)  I don't mind semicolons ... it's that "except when they are necessary" that always gets ya. Generally, whatever we do should be consistent, automatically (GH action) linted and ideally auto-formatted via `pretter` etc.

**Review by walkah — APPROVED** [review]
Mr T Dancing - oVjKPgSf93rJ2GE6Jm

This is awesome! @bgins let's get these PRs merged - we'll work out publishing separately :)

**Pull request merged**